include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -28,7 +27,7 @@ class StatsdClient {
     StatsdClient(const std::string& host,
                  const uint16_t port,
                  const std::string& prefix,
-                 const std::optional<uint64_t> batchsize = std::nullopt) noexcept;
```

Comment:
there were two main changes for c++11 compatibility. the first was removing the use of `std::optional`. I chose to use a sentinel value instead. For the batchsize i just used `max` of `uint64_t` to signal unset.

Reply:
Do you see anything against using `0` as the unset signal?

Reply:
nope, no reason not to use that, sounds good!

Reply:
done!